aws-lc-rs/README.md

```diff
@@ -70,6 +70,30 @@ the pre-generated bindings. This feature requires `libclang` to be installed. Se
 [requirements](https://rust-lang.github.io/rust-bindgen/requirements.html)
 for [rust-bindgen](https://github.com/rust-lang/rust-bindgen)
 
+##### prebuilt-nasm
```

Comment:
I think we should document that the feature flag is honored for all indirect dependencies of the crate that enables it.  We should caution against use without understanding the impact for all dependencies.